api-catalog-ui/frontend/src/epics/fetch-tiles-epic.test.jsx

```diff
@@ -8,15 +8,32 @@
  * Copyright Contributors to the Zowe Project.
  */
 
-import { ActionsObservable } from 'redux-observable';
-import { of, throwError } from 'rxjs';
+import { ofType } from 'redux-observable';
+import { of, throwError, Observable } from 'rxjs';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { TestScheduler } from 'rxjs/testing';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { AjaxError } from 'rxjs/ajax';
 import { fetchTilesPollingEpic } from './fetch-tiles';
 import { fetchTilesFailed, fetchTilesStart, fetchTilesStop, fetchTilesSuccess } from '../actions/catalog-tile-actions';
 
+class ActionsObservable extends Observable {
```

Comment:
What was the reason to create our own class rather then using directly `ActionsObservable` from `redux-observable`?

Reply:
@taban03 `ActionsObservable` was removed from `redux-observable` in v2, this PR upgrades to v2 because otherwise there is a dependency conflict with `rxjs` v7 (and generally upgrading versions was the goal of this PR).

I had trouble getting this test to run without an `ActionsObservable` class, so I created it myself. If you have suggestions for changing the test to not need the class at all, makes sense to do that instead.